reason_4.0/lib/core/minisite_templates/modules/events.php

```diff
@@ -3719,6 +3769,25 @@ function get_ical($events)
   		return $calendar -> get_icalendar_events();
 	}
 	/**
+	 * Get a json representation of a set of events
+	 * @param array $events entities
+	 * @return string JSON
+	 */
+	function get_json($events)
+	{
+		if(!is_array($events))
+		{
+			trigger_error('get_json needs an array of event entities');
+			return '';
+		}
+
+		$events = array_values(array_map(function($e) {
+			return $e->get_values();
+		}, $events));
+
+		return json_encode($events);
```

Comment:
Do you have a preferred place for utility method-ish things? Could/should such a helper go into `/reason_4.0/lib/core/function_libraries`?